kinto/views/flush.py

```diff
@@ -8,9 +8,16 @@
                 path='/__flush__')
 
 
+class ServerFlushed(object):
```

Comment:
That was my initial take (following ``cliquet.events`` paradigm) but I wasn't sure if we needed to create that for just one event. I will move the changes to the new module.